core/src/main/java/org/streamprocessor/core/transforms/DynamodbFn.java

```diff
@@ -97,15 +95,24 @@ public void processElement(@Element PubsubMessage received, OutputReceiver<Pubsu
                         payloadObject.put(
                                 "event_timestamp", dynamodbStreamObject.getString("Published"));
                     } else {
-                        payloadObject.put(
-                                "event_timestamp",
-                                DateTime.now().withZone(DateTimeZone.UTC).toString());
+                    LOG.error(
```

Comment:
Is it enough to just log here or do we need to throw an exception as well? Or perhaps that is already handled if the fields are marked as `REQUIRED` in bigquery?

Reply:
Let's throw instead!